### PR TITLE
include direct plus loans in the total

### PR DESCRIPTION
**Pull request by ascott1**
This adds direct plus loans to the federal loan total. Note that it _does not_ enforce ranges or anything else for these loans, but just adds them to the sum.
## Review

@mistergone or @higs4281

### CODE REVIEW

File: src/loans/index.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ function loanTotals( data ) {
   data.federalTotal = data.perkins +
                       data.staffSubsidized +
                       data.staffUnsubsidized +
+                      data.directPlus +
```
Comment (higs4281) [inline src/loans/index.js:42]: as I read it, "directPlus" loans encompass Plus loans to graduates and Plus loans to parents.
so here, is data.directPlus just representing parental Plus loans? If so, should we call it that?

Comment (ascott1) [inline src/loans/index.js:42]: It represents this bit from the UI:

screen shot 2015-12-15 at 4 32 10 pm

Is that the Plus loan for parents?

Comment (higs4281) [inline src/loans/index.js:42]: I think we need to clarify with Patrick.
Otherwise, :+1:

**Comment by mistergone** [comment]
:+1: Taking it to Mergetown! :city_sunset:

**Pull request merged**